app/src/main/java/pl/valueadd/restcountries/presentation/main/countries/list/CountryListPresenter.kt

```diff
@@ -84,4 +126,11 @@ class CountryListPresenter
 
         it.showError(message)
     }
+
+    internal class Filters {
```

Comment:
Maybe you can use `enum` or `sealed` ?

Reply:
I don't see the need to use `sealed` or `enum`